mcp-server/DEPLOYMENT.md

```
@@ -0,0 +1,304 @@
+railway login
+```
+
+2. Deploy from `mcp-server` directory:
+```bash
+cd mcp-server
+railway init
+railway up
+```
+
+3. Set environment variables in Railway dashboard:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)